consensus/types/src/light_client_header.rs

```diff
@@ -130,8 +130,8 @@ impl<E: EthSpec> LightClientHeader<E> {
 
     pub fn ssz_max_var_len_for_fork(fork_name: ForkName) -> usize {
         match fork_name {
-            ForkName::Base | ForkName::Altair | ForkName::Bellatrix => 0,
-            ForkName::Capella | ForkName::Deneb | ForkName::Electra => {
+            ForkName::Base | ForkName::Altair => 0,
+            ForkName::Bellatrix | ForkName::Capella | ForkName::Deneb | ForkName::Electra => {
```

Comment:
Switch to bellatrix_enabled()

Reply:
In this particular case, I *think* this may be a bad idea. Presumably the fork after `Electra` might include [`EIP-7732`](https://eips.ethereum.org/EIPS/eip-7732). In this EIP (at least at the moment), the `ExecutionPayloadHeader` doesn't exist. Note that what it calls the `ExecutionPayloadHeader` is actually a bid rather than a payload header (working on renaming it). Still working on understanding how light clients work in this model, but either way, our attention needs to be drawn here when we do implement it.

Reply:
aren't these just used in RPC limit calculations? I don't think those are covered in spec tests. this arm of the match statement currently no other variable length fields are added to a light client header which doesn't seem like a safe assumption, so I think mark's right we should consider it every fork